doc/lua_api.md

```diff
@@ -7493,6 +7496,8 @@ An `InvRef` is a reference to an inventory.
 * `is_empty(listname)`: return `true` if list is empty
 * `get_size(listname)`: get size of a list
 * `set_size(listname, size)`: set size of a list
+    * If `listname` is not known, a new list will be created
+    * Setting `size` to 0 deletes a list
```

Comment:
personally I like this better
```suggestion
    * With `size == 0` it deletes a list
```

Reply:
Mm I find mine more straightforward but if it's a problem, feel free to commit the suggestion, it's not a big deal